packages/server-wallet/src/__test__/create-and-close-channel/direct-funding.test.ts

```diff
@@ -93,6 +93,10 @@ it('Create a directly funded channel between two wallets ', async () => {
     turnNum: 0,
   });
 
+  // Let's check what wallet a is waiting for. Expect it to be 'theirPreFundSetup' or ''
+  // (if the objective has never been cranked)
```

Comment:
This is a bit unsatisfactory, but I think easily fixable. When we open a channel via an API call, we do not trigger a crank, so the `waitingFor` doesn't get updated, and takes the current default of `''`. I think we should either 
- trigger the crank (it should be cheap and return early)
- manually patch the `waitingFor` or just do that when we insert the objective. 
- choose a better initial value for the `waitingFor` string.

Reply:
> manually patch the waitingFor or just do that when we insert the objective.

to me, this seems like the natural choice.
- the peer who creates the objective knows what they'll be waiting for
- other peers will often be waiting for `'toBeAccepted'` or something like that. (I don't know what the default should be for auto-accepted objectives.)